### ISSUE TITLE
The screenshots on F-Droid client need to be updated

### ISSUE THREAD

**Issue by SecularSteve** [falzonv/discreet-launcher#259]
It makes this launcher look dated when it's really not.

**Comment by falzonv** [comment]
Hello,

Thank you for the report!

It is quite strange because they are up-to-date in the website (https://f-droid.org/packages/com.vincent_falzon.discreetlauncher/), but indeed not in the F-Droid client...

At the time we did the change, I already noticed that it was not updated in the client even after cleaning my cache but, because it was working properly on the website, I assumed it was related to an F-Droid server cache somewhere and forgot about it.

Just created a ticket at F-Droid, we will see if they have an idea:
https://gitlab.com/fdroid/fdroiddata/-/issues/2842

@Donnnno: did you also noticed that behavior for other apps were screenshots were updated?

Best regards.
